src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/toolWindow/UnrealToolWindowFactory.kt

```diff
@@ -103,38 +185,63 @@ class UnrealToolWindowFactory(val project: Project)
 
     fun print(unrealLogEvent: UnrealLogEvent) {
         val consoleView = UnrealPane.currentConsoleView
+        var startOfLineOffset = consoleView.contentSize
```

Comment:
Looks like the fix for exception in createHyperlink was reverted

Reply:
It might be an old branch, let's check after rebase